python/tests/test_run_metta.py

```diff
@@ -48,3 +48,33 @@ def test_list_concatenation(self):
         result = MeTTa().run(program)
         self.assertEqual('[[(Cons a1 (Cons a2 (Cons b1 (Cons b2 Nil))))]]', repr(result))
 
+    def test_comments(self):
```

Comment:
As changes in functionality are made in Rust code, we need adding Rust unit tests to check them. Otherwise other platforms are not covered. Could you migrate tests to Rust please.